shared/js/download/download_ui.js

```diff
@@ -224,16 +223,14 @@ var DownloadUI = (function() {
   }
 
   function addActionMenu() {
+    // If we have an actionMenu, empty it and re-create it.
     if (actionMenu !== null) {
       actionMenu.innerHTML = '';
```

Comment:
I think if you're removing the actionMenu, there's no need to set innerHTML empty.